js/main.js

```diff
@@ -137,4 +137,23 @@ function pupUpMenuid(id) {
 const viewBtn = document.getElementsByClassName('popup-window');
 for (let i = 0; i < viewBtn.length; i += 1) {
   viewBtn[i].addEventListener('click', () => pupUpMenuid(i));
-}
\ No newline at end of file
+}
+
+const contactFormEmail = document.querySelector('form #email');
+const contactFormForm = document.querySelector('form button');
+
+contactFormForm.addEventListener('click', (event) => {
+  if (contactFormEmail.value !== contactFormEmail.value.toLowerCase()) {
+    event.preventDefault();
+    document.querySelector('form .erorr-massege').classList.add('active');
+  } else {
+    document.querySelector('form .erorr-massege').classList.remove('active');
+  }
+});
+contactFormEmail.addEventListener('input', () => {
+  if (contactFormEmail.value !== contactFormEmail.value.toLowerCase()) {
+    document.querySelector('form label .erorr-massege').classList.add('active');
```

Comment:
* According to the [project requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#:~:text=When%20the%20user,is%20not%20sent.) the user clicks submit form button before the email is validated and an error message displayed but currently yours displays the error message  shows up before the submit button, Kindly fix this 🙏

Reply:
@Davidosky007 Is it good now?

Reply:
@Davidosky007 why you didn't say that in your first review I don't have an extra review and not I only have one review